tcms/core/tests/test_ajax.py

```diff
@@ -91,3 +95,87 @@ def test_update_default_tester_non_existing_user(self):
         self.assertEqual(result['response'], 'Default tester not found!')
 
         self._assert_default_tester_is(None)
+
+
+class Test_Tag(BasePlanCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.url = reverse('ajax-tags')
+        cls.test_tag = TestTag.objects.create(name='test_tag')
```

Comment:
Try to use factories for test data creation and try not using hard-coded names unless strictly required. Factories will use the next available name if the one is taken. objects.create will most likely fail with unique constraint errors instead.